source/java-drivers.txt

```diff
@@ -20,11 +20,16 @@ Java MongoDB Drivers
 Introduction
 ------------
 
-`Java Driver <https://docs.mongodb.com/drivers/java/sync/current>`__ is the recommended MongoDB Java Driver.
-
-For asynchronous stream processing and reactive streams interoperability,
-the :doc:`Reactive Streams Driver </reactive-streams>` is the
-recommended MongoDB Java Driver.
+Welcome to the documentation site for the official MongoDB Java 
+drivers. You can add one of the following drivers to your application 
+to work with MongoDB in Java:
+
+- Use the 
+  `Java Driver <https://docs.mongodb.com/drivers/java/sync/current>`__ 
+  for synchronous Java applications.
+  
+- Use the :doc:`Reactive Streams Driver </reactive-streams>` for 
+  asynchronous stream processing and reactive streams interoperability.
```

Comment:
I would probably describe this as "to use the Reactive Streams API for asynchronous stream processing" because "Reactive Streams" is a framework.

Reply:
Using your words